src/components/contributors-list.js

```diff
@@ -54,19 +54,26 @@ export default function ContributorsList({
     }
 
     return (
-      <>
+      <div data-testid="contributors-list-inline">
         <FormattedMessage id='ui-eholdings.label.contributors' />
         &#58;&nbsp;
         <span data-test-eholdings-contributors-inline-list-item>
           {contributorsJointList}
         </span>
-      </>
+      </div>
     );
   };
 
   return showInline ? showInlineList(contributorsInlineLimit) : showKeyValueList();
-}
+};
 
 ContributorsList.propTypes = {
-  data: PropTypes.array
+  contributorsInlineLimit: PropTypes.number,
+  data: PropTypes.arrayOf(PropTypes.shape({
+    contributor: PropTypes.string,
+    type: PropTypes.string,
+  })).isRequired,
+  showInline: PropTypes.bool,
```

Comment:
If this prop is not required, better to add it's default value
```
ContributorsList.propTypes = { showInLine: false };
````
Also check `contributorsInlineLimit`